extensions/sql-database-projects/package.json

```diff
@@ -234,82 +239,82 @@
       "view/item/context": [
         {
           "command": "sqlDatabaseProjects.build",
-          "when": "view == sqlDatabaseProjectsView && viewItem == databaseProject.itemType.project",
+          "when": "view =~ /^(sqlDatabaseProjectsView|dataworkspace.views.main)$/ && viewItem == databaseProject.itemType.project",
```

Comment:
I had no idea we could do this with regex; neat!  Is there any reason to keep sqlDatabaseProjectsView around with the introduction of the dataworkspace view?

Reply:
as we discussed the other day, we will be having these 2 views co-exist for a while.